packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -267,6 +267,7 @@ class _FloatingActionButtonState extends State<FloatingActionButton> {
       onHighlightChanged: _handleHighlightChanged,
       elevation: _highlight ? widget.highlightElevation : widget.elevation,
       constraints: widget._sizeConstraints,
+      slopPadding: widget.mini ? const EdgeInsets.all(4.0) : null,
```

Comment:
Might be worth documenting this side effect on the `mini` property.

Reply:
Done